### DISCUSSION TITLE
Request body format to healthchecks

### DISCUSSION THREAD

**Discussion by nesalkiN** [garethgeorge/backrest#546] [Q&A]
The request body for calls to healthchecks looks like this:
![Capture_Backrest_-_Immich_–_Healthchecks io_–_Brave_2024-11-10_16-53-48_86046533](https://github.com/user-attachments/assets/b43bcb4e-1bb1-41b8-bbb1-babcc0b3b224)

This is quite hard to read. The warning via e-mail and notification services connected to healthchecks shows this info when it sends out warnings but you can't easily see in the notifications what's going on when in json.

Can this be configured in the field for template? Please ELI5 if possible. ;)

For other software/solutions it could look like this: (this is from duplicati):
![Capture_Duplicati_-_Appdata_OneDrive_–_Healthchecks io_–_B_2024-11-10_16-49-07_62998342](https://github.com/user-attachments/assets/d909e858-47d9-4af4-ae7c-e7d864d98dc7)

(pics from healthchecks)

**Comment by inode64** [comment]
When I made the pull request I did not pay attention to this detail.
The option to send in html format would have to be added, because by default all hooks send the message as json.

I just have a little time, I'll prepare it.
